source/main/environment.js

```diff
@@ -28,9 +29,13 @@ const isTestnet = NETWORK === TESTNET;
 const isDevelopment = NETWORK === DEVELOPMENT;
 const isWatchMode = process.env.IS_WATCH_MODE;
 const API_VERSION = process.env.API_VERSION || 'dev';
+const mainProcessID = process.ppid;
```

Comment:
@DeeJayElly you can probably solve the flow issue by doing something like this:

```
const mainProcessID = process.ppid || '-';
```

Reply:
Fixed with 

const mainProcessID = get(process, 'ppid', '-');